### DISCUSSION TITLE
Centering website title in navigation bar of website

### DISCUSSION THREAD

**Discussion by bellescoincidences** [quarto-dev/quarto-cli#9420] [Q&A]
### Description

Hello!

I am doing front-end website design for my team to use (internally only), using Quarto ©. I am working off prototypes that have been developed using agile methods (user research, requirements gathering etc). On top of this, this is an internal public sector website that has some specific user-based requirements (ie. accessibility, formatting). 

I am customising the styling of the pages using, where possible, a custom .scss stylesheet, and trying to **not** use the _quarto.yml but I have run into difficulty getting the .css to work where it otherwise would in standard (native) frontend HTML/CSS/JavaScript frontend development. I have spent literally hours (30 hours plus) trying to answer the following questions. 😅 So now, finally, I am reaching out to you for your help; at least I hope you may be able to give indication of _feasability_, so that I don't have to spend hours _trying_ reverse engineering.

**1. How can I centre the page navbar title, with menu list items on both the left and right?
2. Is it possible to create a drop-down menu on the navigation bar (navbar) that is a **sidebar** on the page? ie, a drop-down that triggers a collapsable/ sidebar?**

I am aware Quarto © uses HTML partials, and had a quick look through it, but frankly it's all a bit gibberish to me. 

Any guidance you can give me would be much appreciated! 

NB: Unfortunately, I cannot share the repo with you as it is internal and, therefore, private. I don't think it would add much anyway given that my _quarto.yml file is very standard and, as I said, I am only using it for the rendering bit, where I can. I can probably share the .scss with you if needed.

**Comment by cscheid** [comment]
Sorry, we unfortunately really cannot help you if you don't provide specifics.

**Reply by mcanouil** [reply]
@bellescoincidences Maybe you could work on a small website with little modification using `quarto create project website`. Be sure to also read <https://quarto.org/docs/output-formats/html-themes.html>.
